src/sentry/utils/kvstore/bigtable.py

```diff
@@ -116,7 +117,10 @@ def _get_table(self, admin: bool = False) -> Table:
 
     def get(self, key: str) -> bytes | None:
         with sentry_sdk.start_span(op="bigtable.get"):
-            row = self._get_table().read_row(key)
+            # Default timeout is 60 seconds, much too long for our ingestion pipeline
+            # Modify retry based on https://cloud.google.com/python/docs/reference/storage/latest/retry_timeout#configuring-retries
+            modified_retry = DEFAULT_RETRY_READ_ROWS.with_deadline(5.0)
```

Comment:
TODO: change to with_timeout instead of with_deadline, per GCP employee recommendation.